builtin/providers/test/resource_with_custom_diff.go

```diff
@@ -0,0 +1,84 @@
+package test
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func testResourceCustomDiff() *schema.Resource {
+	return &schema.Resource{
+		Create:        testResourceCustomDiffCreate,
+		Read:          testResourceCustomDiffRead,
+		CustomizeDiff: testResourceCustomDiffCustomizeDiff,
```

Comment:
How about `Review` or `Revise`? That is kind of what is happening here... the diff is being reviewed, and possibly being vetoed or rejected. :)

Other names I could think of were either too close to CRUD-like things that already exist (like `Patch`) or just didn't really roll off the tongue like the current set of functions.